README.md

```diff
@@ -93,19 +93,20 @@ Note that states and issue transitions are based on your Jira project workflow c
 
 #### Step 2: Configure webhooks in Jira
 
-If you want to [send notifications from Jira to Mattermost](#11-send-notifications-from-Jira-to-Mattermost), link a Jira project to a Mattermost channel via webhooks.
+As of Jira 2.1, the plugin gets sent a firehose of events from the server via the webhook below.  Previously configured webhooks that point to specific channels are still supported and will continue to work. 
+
+To control Mattermost channel subscriptions use the command `/jira subscribe` in the channel where you want to receive subscriptions.  It will open a new modal window to select the project and event triggers that will post to the channel. 
```

Comment:
Comma before "use" in the first sentence

Not sure if this reads better but:
> in the channel ~~where~~ **in which** you want to receive subscriptions

Also "receive subscriptions" sounds a but odd. Subscription is something you configure, as opposed to something like receiving an "event" or "message".